config/manager/manager.yaml

```diff
@@ -37,6 +38,8 @@ spec:
             value: quay.io/netobserv/network-observability-console-plugin:v0.1.11
           - name: DOWNSTREAM_DEPLOYMENT
             value: "false"
+          - name: PROFILING_BIND_ADDRESS
```

Comment:
why not call it PROFILE_PORT since what we pass here is is just the port like we do in the agent and pls add profile md file to doc with examples of the different collections with HOWTOs

Reply:
the operator already uses this kind of addresses (e.g. for its metric server) so I prefer to be consistent within this repo 
I've added some doc in DEVELOPMENT.md